sender-app/configuration/application.properties

```diff
@@ -61,6 +61,10 @@ logging.level.sender-complex-obs-sync=${openmrs.eip.log.level}
 
 # ****************************** Actuator Configuration ****************************************************************
 #
+# To prevent errors at startup with missing class: META-INF/services/org/apache/activemq/transport/vm
+# because of overlaps between ActiveMQ and Artemis.
+spring.autoconfigure.exclude=org.springframework.boot.actuate.autoconfigure.jms.JmsHealthContributorAutoConfiguration
```

Comment:
Since this is a configurable thing, don't you think we should keep it out of the codebase? Otherwise other implementations will need the health checker.

If the property doesn't work and this is a blocker may be in the interest of time we might just merge this as is otherwise I suggest we do the following for a future release,

I want to think this is only happening for the sender and not receiver which explains why you're actually only adding it to the sender's properties file, this is because in the sender we never define a connection factory with a broker URL with proper transport like we do in the receiver, so I think the fix is to add a connection factory bean to the sender too that explicitly uses another transport and not vm.

I have also created [EIP-125](https://issues.openmrs.org/browse/EIP-125) to fix openmrs-eip in the future too because it unnecessarily ships with activemq libraries.

What do you suggest @mks-d and @icrc-fdeniger?

Reply:
I added this property because I believed it will fix the unit test issues on branch 3.0.x
But yes maybe if we remove the activemq libraries we won't have this issue.
For other applications, I believe that in any case, they will have their own application.properties file.

Reply:
The tests still fail for me even with your change, note that this change is in a file that is not on the classpath, so it has no effect on unit tests

Reply:
The failing test are `OpenmrsLoadProducerTest`, `OpenmrsLoadProducerIntegrationTest` and HashBatchUpdaterTest and their issue something else that is not related